### ISSUE TITLE
Create your own cards

### ISSUE THREAD

**Issue by KevinBongart** [KevinBongart/cards_against_humanity#25]

**Comment by owenwastaken** [comment]
Is it already possible to add cards by amending them to /lib/cards/full.json?

**Comment by KevinBongart** [comment]
Yes, that's how I've added the official (open source) pack to the app.

I don't remember exactly the context when opening this issue, but I think it was supposed to be a feature request so that players could add their own cards when starting a new game (unlike adding cards to the JSON file, which requires a PR, a deploy and a data migration).
